tests/unit/swagger/schemaDetails3.spec.ts

```diff
@@ -698,6 +698,30 @@ describe('Definition generation for OpenAPI 3.0.0', () => {
             },
           });
         });
+
+        it('Supports custom example labels', () => {
```

Comment:
Can you please add specs for OpenAPI 2? While that version does not support examples/example, it still should be clear what happens. In other cases, we add this as an `x-example`, I think it would be nice to add the first example without label and possibly warn about the lack of support.

Reply:
I get what you mean in general but not sure about the "example without label". I used an example with label in test and emit a warning if exampleLabels is not empty which should indicate usage of example labels.